core/eth/reader.go

```diff
@@ -288,6 +291,7 @@ func (t *Reader) GetRegisteredQuorumIdsForOperator(ctx context.Context, operator
 func (t *Reader) getRegistrationParams(
 	ctx context.Context,
 	keypair *core.KeyPair,
```

Comment:
Do we still need keypair?

Reply:
we don't actually - I think I kept it around while refactoring. I can remove it.

Reply:
2e43636e769b61079858a316134ecdb5e508b517